### PR TITLE
AAP-33062 - add more details for LDAP Group types

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR addresses the changes requested in https://issues.redhat.com/browse/AAP-33062 to provide more details regarding LDAP Group types.

### CODE REVIEW

**Review by sayjadha — COMMENTED** [review]
@dcdacosta : A few suggested edits, rest LGTM.

File: downstream/modules/platform/proc-controller-set-up-LDAP.adoc [hunk 1]
```diff
@@ -17,32 +17,49 @@ Users created through an LDAP login should not change their username, first name
...
++
+----
+{"name_attr": "cn", "member_attr": "member"}
+----
 +
 To determine the parameters that a specific *LDAP Group Type* requires, refer to the link:https://django-auth-ldap.readthedocs.io/en/latest/reference.html#django_auth_ldap.config.LDAPGroupType[django_auth_ldap documentation] on the classes `init` parameters.
 +
 . In the *LDAP Group Search* field, specify which groups should be searched and how to search them. 
-. In the *LDAP User Attribute Map* field, enter user attributes to map LDAP fields to your Ansible Automation Platform users for example, email or first_name. 
+. In the *LDAP User Attribute Map* field, enter user attributes to map LDAP fields to your {PlatformNameShort} users for example, email or first_name.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-LDAP.adoc:62]: Fixed.

File: downstream/modules/platform/proc-controller-set-up-LDAP.adoc [hunk 2]
```diff
@@ -17,32 +17,49 @@ Users created through an LDAP login should not change their username, first name
...
 . *LDAP Start TLS* is disabled by default. To enable TLS when the LDAP connection is not using SSL, set the switch to *On*.
 +
 include::snippets/snip-gw-authentication-additional-auth-fields.adoc[]
 +
 . Enter any *LDAP Connection Options* to set for the LDAP connection. 
-. In the *LDAP Group Type Parameters* field, enter the parameters required for the LDAP Group Type you previously selected to identify LDAP groups and the members that belong to those groups.
+. Depending on the selected *LDAP Group Type*, different parameters are available in the *LDAP Group Type Parameters* field to account for this. LDAP_GROUP_TYPE_PARAMS is a dictionary, which is converted to kwargs and passed to the *LDAP Group Type* class selected. There are two common parameters used by the LDAP group types; `name_attr` and `member_attr`. Where `name_attr` defaults to `cn` and `member_attr` defaults to `member`:
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-LDAP.adoc:53]: Fixed by changing the plural to generic "group types" in the last sentence.

[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-LDAP.adoc:53]: Fixed.

File: downstream/modules/platform/proc-controller-set-up-LDAP.adoc [hunk 3]
```diff
@@ -17,32 +17,49 @@ Users created through an LDAP login should not change their username, first name
...
+* `NestedGroupOfUniqueNamesType`
+* `NestedActiveDirectoryGroupType`
+* `NestedOrganizationalRoleGroupType`
+* `NestedMemberDNGroupType`
+* `PosixUIDGroupType`
 +
 [NOTE]
 ====
-The LDAP group types that are supported by the Ansible Automation Platform use the underlying link:https://django-auth-ldap.readthedocs.io/en/latest/reference.html#django_auth_ldap.config.LDAPGroupType[django-auth-ldap library]. 
+The *LDAP group types* that are supported by the {PlatformNameShort} use the underlying link:https://django-auth-ldap.readthedocs.io/en/latest/reference.html#django_auth_ldap.config.LDAPGroupType[django-auth-ldap library]. To specify the parameters for the selected group type, see Step 13 of this procedure.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-LDAP.adoc:45]: The field is LDAP Group Type. Changed to generic group types for plural instance in this case.

**Review by sayjadha — APPROVED** [review]
The changes LGTM.

**Pull request merged**